src/includes/set-fingerprint/rpc/apple.mdx

```diff
@@ -0,0 +1,39 @@
+```swift
+import Sentry
+
+SentrySDK.start { options in
+    options.beforeSend = { event in
+        if let error = event.error as NSError? {
+            if let failingUrl = error.userInfo[NSURLErrorFailingURLErrorKey] as? String,
+               error.domain == NSURLErrorDomain {
+                event.fingerprint = [
+                    "{{ default }}",
+                    failingUrl,
+                    String(error.code),
+                ]
+            }
+        }
+        return event
+    }
+}
+```
+
+```objc {tabTitle:Objective-C}
+@import Sentry;
+
+[SentrySDK startWithConfigureOptions:^(SentryOptions *options) {
+    options.beforeSend = ^SentryEvent * _Nullable(SentryEvent * _Nonnull event) {
+        if ([event.error.domain isEqualToString:NSURLErrorDomain]) {
+            NSString *failingUrl = event.error.userInfo[NSURLErrorFailingURLErrorKey];
+            if (failingUrl != nil) {
+                event.fingerprint = @[
+                    @"{{ default }}",
+                    failingUrl,
+                    @(event.error.code).stringValue
```

Comment:
L: 
```suggestion
                    [NSString stringWithFormat:@"%i",event.error.code];
```

This is me nit picking, but you are boxing an integer in a NSNumber, and then converting it to string, when you could do it directly.